suite-common/message-system/config/config.v1.json

```diff
@@ -1,8 +1,70 @@
 {
     "version": 1,
-    "timestamp": "2024-01-03T00:00:00+00:00",
+    "timestamp": "2025-01-06T00:00:00+00:00",
```

Comment:
@matejkriz could you please advise if this message-system "action" is properly defined?